onadata/libs/serializers/project_serializer.py

```diff
@@ -369,6 +369,15 @@ def validate(self, attrs):
             })
         return attrs
 
+    def validate_public(self, value):  # pylint: disable=no-self-use
+        """
+        Validate the public field
+        """
+        if not settings.ALLOW_PUBLIC_DATASETS and value:
+            raise serializers.ValidationError(
+                'Public projects are currently disabled.')
```

Comment:
Can you please enable translation support? 
```python
_("Public projects are currently disabled.")
```

Reply:
Yes, let me change that